client/web/src/enterprise/batches/create/CreateOrEditBatchChangePage.tsx

```diff
@@ -271,6 +273,12 @@ const EditPage: React.FunctionComponent<EditPageProps> = ({ batchChange, refetch
     )
 
     const [filters, setFilters] = useState<WorkspacePreviewFilters>()
+    const [isDownloadSpecModalOpen, setIsDownloadSpecModalOpen] = useState<boolean>(false)
```

Comment:
It doesn't really matter, but I don't think you need the explicit type here. This _should_ also work:

```suggestion
    const [isDownloadSpecModalOpen, setIsDownloadSpecModalOpen] = useState(false)
```

(You don't need to actually apply this change; just an FYI for future work.)